lib/modules/manager/html/readme.md

```diff
@@ -1 +1,7 @@
-The `html` manager extracts scripts and CSS includes pointing to cdnjs. It differs from the `cdnjs` manager in that it detects if an integrity hash is already present and updates it too if found.
+The `html` manager updates `<script>` tags and _CSS includes_ that point to the [cdnjs content delivery network](https://cdnjs.com/).
```

Comment:
Just saying _CSS includes_ is too vague. What's the correct CSS syntax?

Reply:
We don't really care about the syntax - we just look for cdnjs URLs

Reply:
If we don't care about the syntax, then we don't need to mention it. 😄